### DISCUSSION TITLE
never mind, JLPCB print services will not work. (original title: frame STL error on JLPCB. ok to bypass?)

### DISCUSSION THREAD

**Discussion by mer2329** [sanni/cartreader#510] [Need help]
never mind JLCPCB pints wont work, for a different reason

hi, im ordering the frame through JLCPCB and its getting flagged with 1 error "0.8mm wall thickness detected"
would i be ok confirming this or will this cause failure.
![image](https://user-images.githubusercontent.com/6768243/189421360-bfc7b2e6-355c-40dd-b881-031f8db438ff.png)
![image](https://user-images.githubusercontent.com/6768243/189421451-256e70ff-2f9e-473b-b81e-f8a6ac46cf0d.png)
im using this file for the upload https://github.com/sanni/cartreader/blob/master/hardware/main_pcb_hw5/Main%20PCB%20Stand%20-%20Left%20N64%20Adapter.stl

**Comment by mer2329** [comment]
NVM just figured out SLA pints wont work as resin will not melt for the heat set inserts.

**Reply by sanni** [reply]
Good point, I didn't know that. Now that resin printing is actually cheaper than FDM I will keep that in mind for any future case design.

**Reply by mer2329** [reply]
one question i was wondering is why not use captive nuts for the threads. personally ive never had issue with that, it is more complicated to design with that in mind, but thankfully metric nuts have a highly standardized size. 
see this thing i made in the past for my take on captive nuts https://www.thingiverse.com/see this thing i made in the past for my take on captive nuts https://www.thingiverse.com/thing:2769181

but rather then captive nuts, with the current design couldn't the brass female-female standoffs be used to secure the PCB to the frame? so a simple 2.5mm hole would be perfect for that. i have both male-female, and female-female standoffs/spacer and the screw on the male-female standoff just isn't long enough to work well through the PCB
and if you use a hex socket screw it should be very easy to design for. and a washer could be used to reinforce it.

**Comment by Ancyker** [comment]
I got mine printed in Nylon (MJF). It's the same price as resin.

**Reply by mer2329** [reply]
not on JLCPCB
right now the price for black resin is $3.91 per part and the price for black nylon is $8.92 per part

**Comment by redchairlabs** [comment]
I'm cracking open an old thread but I just tried to order this from JLC 3DP and received the same warning message as above. They are suggesting increasing wall thickness to 1.2mm minimum or idealy 2mm. It appears to be at the areas of the threaded inserts.

I ordered mine FDM with PLA as recomended here: [github.com/sanni/cartreader/wiki/What-to-order#3d-print](https://github.com/sanni/cartreader/wiki/What-to-order#3d-print)

Has anyone had a successfull print with the existing thickness?

**Reply by Ancyker** [reply]
I ignored it.

![image](https://github.com/sanni/cartreader/assets/7105333/2ca1ef0f-61cb-4ca3-95e6-da844039ba85)

![image](https://github.com/sanni/cartreader/assets/7105333/b8a6a580-a67d-4a5f-86c7-bdc2d1e0a335)

I ordered them in nylon.

**Reply by redchairlabs** [reply]
I ignored it too. How did the nylon turn out?

**Reply by Ancyker** [reply]
Good. My current frame is PLA though.

**Reply by mer2329** [reply]
i wound up ordering 5 frames off shapeways, i got black PLA. 
i also ordered proper heat set tips for my iron. so the first frame i heat set it. it was HORRIBLE. the heat sets wound up skewed. and the display mount points were horribly mangled.

for the other 4 i built i use 10mm m2 pan head screws into double female standoffs. i also used 2 washers between bottom PCB and standoff. i also added a washer under the screw head for more even pressure. 
for the display i used m2 8mm pan head screws no washers
this setup has been great. no issues, and better yet, no deformed frames.

**Reply by redchairlabs** [reply]
Glad it worked out. i should find a heat set tip for my iron then.

**Reply by mer2329** [reply]
i have a hakko FX 888d and these fit perfectly https://www.amazon.com/Virtjoule-Heat-Insert-Tips-Sizes/dp/B08B17VQLD

**Reply by redchairlabs** [reply]
Awesome! I have an old Weller  SP40N and it looks lke these will fit it. Thanks for sharing, saved me some money.

**Reply by mer2329** [reply]
glad i saved you some money on that. but remember my warning on the inserts, they didnt work well for me, def order an extra frame so you can test and see if you like them

**Reply by redchairlabs** [reply]
I ordered 5 frames just in case I messed one up or break a few.

I was wondering, would it be possible to thru bolt instead of using the inserts?

**Reply by mer2329** [reply]
thats what i wound up doing. for each through hole i did the following
note: all screws do not contain the head in the measurement (as thats only dont for countersunk screws). all measurements are taken with my digital caliper

for the main assembely, all screws are is m2 with the same threading. 
pan head screw 10mm > 1x washer .3mm thick > frame > bottom PCB > 3x washers .9mm thick > 18mm m2 female/female brass sandoff > top PCB > socket screw 6mm long

for the display i simply used used m3 screws 8mm long pan head screws. but 6mm screws will work too, also note 10mm screws work, but 1mm of them will poke out the top 2 holes.

because i buy these screws in bulk (100 count minimum) and buy stainless steel screws. i buy from albany county fasteners due to great prices when you need quantity. https://www.albanycountyfasteners.com/
the brass standoffs were bought in a kit off ebay, make sure its 2mm standoffs as 2.5mm does exist. 

heres links to the screws i used from albany county fasteners.
m2 socket (i bought this cause i like how they look. but you can use m2 pan head instead to save some money) 8 per frame: https://www.albanycountyfasteners.com/Socket-Head-Cap-Screw-2MM-A2-Stainless-Steel-p/5020000.htm
m2 panhead 8 per frame (16 per frame if you opted to skip m2 socket): https://www.albanycountyfasteners.com/Phillips-Pan-Head-Machine-Screw-M2-x-4-p/1066-1002.htm
m2 washers 32 washers per frame: https://www.albanycountyfasteners.com/Flat-Washers-A2-Stainless-Steel-p/5620000.htm
m3, i used these cause i had them on hand. https://www.albanycountyfasteners.com/Phillips-Pan-Head-Machine-Screw-M3-x-5-p/1066-1008.htm

if your just building 1 frame, just compare prices of the fasteners off Ebay, a store near you that has a good selection of metric fasteners at reasonable prices and albany county fasteners. if your building 5 frames like i did, and you like stainless steel fasteners its hard to beat albany county fasteners prices.

**Reply by redchairlabs** [reply]
Thank you for all that information.
